plugins/org/README-alpha.md

```diff
@@ -0,0 +1,462 @@
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    # use false as value for disabling the extension and true for enabling
+    - entity-card:org/user-profile: false
+    # or
+    # - entity-card:org/user-profile:
+    #   - config:
+    #       # set 'true' for enabling it again
+    #       disabled: true
+```
+
+##### Config
+
+There is only one configuration available for this entity card extension, which is setting an entity filter that determines when the card should be displayed on the entity page.
+
+Here is an example showing the `user-profile` overview card only for entities of kind group or user:
+
+```yaml
+# app-config.yaml
+# example setting the extension to only show up for entities with kind "user"
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    - entity-card:org/user-profile:
+        config:
+          # The default value is "kind:user"
+          # For more information about entity cards filters, check out this pull request
+          # https://github.com/backstage/backstage/pull/21480
+          filter: 'kind:user'
+```
+
+##### Override
+
+Use extension overrides for completely re-implementing the user-profile entity card extension:
+
+```tsx
+import { createExtensionOverrides } from '@backstage/backstage-plugin-api';
+import { createEntityCardExtension } from '@backstage/plugin-catalog-react/alpha';
+
+export default createExtensionOverrides({
+  extensions: [
+    createEntityCardExtension({
+      // These namespace and name are necessary so the system knows that this extension will override the default 'user-profile' entity card extension provided by the 'org' plugin
+      namespace: 'org',
+      name: 'user-profile',
+      // By default, this card will show up only for groups or users
+      filter: 'kind:user'
+      // Returing a custom card component
+      loader: () =>
+        import('./components').then(m => <m.MyCustomOwnershipEntityCard />),
+    }),
+  ],
+});
+```
+
+For more information about where to place extension overrides, see the official [documentation](https://backstage.io/docs/frontend-system/architecture/extension-overrides).
+
+### My Groups Sidebar Item
+
+As the [NavItem](https://backstage.io/docs/reference/frontend-plugin-api.createnavitemextension) extension type does not support conditional rendering, this plugin does not provide navigation items, so to use the `MyGroupsSidebarItem` component, we recommend overriding the [App/Nav](https://backstage.io/docs/frontend-system/building-apps/built-in-extensions#app-nav) extension and adding the item statically.
+
+> [!IMPORTANT]
+> As you can see in the example below, we are using the same attachment point, inputs and outputs as the default App/Nav extension to avoid side effects on the NavItem and NavLogo extensions.
+
+```tsx
```

Comment:
very unfortunate to have to repeat this all over the place, it's so much code and it's subject to change - i wonder if there's a single place we can put it instead, like a "common patterns" section or "guides" or whatever

Reply:
Good point 💡 . I added the example because people who used this custom sidebar item before will probably ask about it... But I think it's fine to keep it here for now as we will be revisiting the sidebar features, so perhaps the override will no longer be necessary.